src/components/02-buttons/button.jsx

```diff
@@ -1,60 +1,83 @@
 {
-  /* 
+  /**  
   ATTRIBUTES FOR THE BUTTON
   ******************************
-  class
-  This will specify what CSS the button will get from USWDS' CSS files.
+  @namespace
+  @property {string} class
+   - This will specify what CSS the button will get from USWDS' CSS files.
   The base class is usa-button with additions coming from the config.class property passed in
   ----------------------------
-  autofocus 
-  This Boolean attribute specifies that the button should have input focus when the page loads. 
+  @property {boolean} autofocus 
+   - This Boolean attribute specifies that the button should have input focus when the page loads. 
   Only one element(button) in a document can have this attribute. 
   ----------------------------
-  disabled
-  Boolean attribute that prevents the user from interacting with a button i.e.(cannot click on the button).
+  @property {boolean} disabled
+   - Boolean attribute that prevents the user from interacting with a button i.e.(cannot click on the button).
   ----------------------------
-  name
-  The name of the button.
+  @property {string} name      - The name of the button.
   ----------------------------
-  type
+  @property {string} type      - 
   submit: The button submits the form data to the server (default type if not specifed). 
   button: Used for events (onclick, onfocus, onhover)
   reset: reset controls to their initial values
   ----------------------------
-  value
-  Defines the value associated with the button’s name when it’s submitted with the form data. This value is passed to the server in params when the form is submitted.
+  @property {string} value
+   - Defines the value associated with the button’s name when it’s submitted with the form data. This value is passed to the server in params when the form is submitted.
   ------------------------------
+  @property {string} innerText  - The text to displayed on the button 
   Setting Up Config 
   ******************************
   single class usage
-  ------------------------------
+  @example 
   const config = {
-    class: "usa-button--secondary",
```

Comment:
`usa-button` is necessary when defining `const config` because in our button element the class attribute defines `usa-button` which is the default style sheet then all other `usa-button--variants` are defined after the default `class = "usa-button usa-button--secondary"` .
[uswds-button](https://designsystem.digital.gov/components/button/)

Reply:
I see thanks for the link my bad Lamarr I'll fix it